pkg/ingester/ingester_test.go

```diff
@@ -1788,6 +1788,14 @@ func TestIngester_Push(t *testing.T) {
 
 	for testName, testData := range tests {
 		t.Run(testName, func(t *testing.T) {
+			memorySeriesStats.Set(0)
+			memoryTenantsStats.Set(0)
+			tenantsWithOutOfOrderEnabledStat.Set(0)
+			minOutOfOrderTimeWindowSecondsStat.Set(0)
+			maxOutOfOrderTimeWindowSecondsStat.Set(0)
+			appendedSamplesStatsBefore := appendedSamplesStats.Total()
+			appendedExemplarsStatsBefore := appendedExemplarsStats.Total()
```

Comment:
Alternatively you may export `Counter.reset()`. I think I didn't export it because counters should be never reset, but we already have `GetAndReset()` so it doesn't look a big deal.

Reply:
I think I'd rather avoid exporting reset.